readme.md

```diff
@@ -12,9 +12,23 @@
 - If your bot is brand new, you will need to wait up to 24h for your bot to be fully registered as "active"
 - Once you are eligible, get the badge at the [Application site](https://discord.com/developers/active-developer)
 
+## How to use on your very own computer wow wow
+- Install [Node.js](https://nodejs.org/en/)
+- Create an application at the [Developer panel](https://discord.com/developers/applications/) (or you can use a previously existing one)
+- Make it into a bot if it isn't already one
+- Obtain your token
+- If you have git installed, clone this repo, otherwise download the zip file from the funny little green code button
+- Extract to a folder
+- Type "cmd" in the address bar at the top of your windows file explorer to open a console, and type "npm run run BOT_TOKEN_HERE"
```

Comment:
are we just assuming noobs use winddows

Reply:
Yes, as it is the most used OS in the world.